### PR TITLE
Change CHANGELOG for 0.3.0

### PR DESCRIPTION
**Pull request by squaremo**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,25 @@
+## 0.3.0 (2017-05-03)
+
+Support for newer Kubernetes.
```
Comment (paulbellamy) [inline CHANGELOG.md]: which?

Comment (squaremo) [inline CHANGELOG.md]: Good point, I'll be more specific.

File: site/installing/standalone.md [hunk 2]
```diff
@@ -52,6 +52,22 @@ runs the Flux service and the Flux daemon in a single pod.
...
+You will need to provide fluxd with a service account which can access
+the namespaces you want to use Flux with. To do this, consult the
+example service account given in
+[flux-account.yaml](../../deploy/standalone/flux-account.yaml) (which
+puts essentially no constraints on the account) and the
+[RBAC documentation](https://kubernetes.io/docs/admin/authorization/rbac/),
+and create a service account in whichever namespace you put fluxd
+in. You may need to alter the `namespace: default` lines, if you adapt
+the example.
+
```
Comment (abuehrle) [inline site/installing/standalone.md]: You must provide `fluxd`  with a service account that can access
 the namespaces you want to use Flux with. Consult the
 example service account in  [flux-account.yaml](../../deploy/standalone/flux-account.yaml) (which has no constraints on the account) and the
 [RBAC documentation](https://kubernetes.io/docs/admin/authorization/rbac/),
 and then create a service account in the namespace that `fluxd` is running. Note that  the `namespace: default` lines may need to be altered, if you adapt
this example.

File: site/installing/standalone.md [hunk 3]
```diff
@@ -52,6 +52,22 @@ runs the Flux service and the Flux daemon in a single pod.
...
+You will need to explicitly tell fluxd to use that service account by
+uncommenting and possible adapting the line `# serviceAccountName:
+flux` in the file `fluxd-deployment.yaml` before applying it.
+
```
Comment (abuehrle) [inline site/installing/standalone.md]: You will need to explicitly tell `fluxd` to use that service account by
 uncommenting and possibly adapting the line `# serviceAccountName:
 flux` in the `fluxd-deployment.yaml` file before applying it.

[i'm missing some context but what service account?]

**Pull request merged**